doc/user/installing-gems.md

```diff
@@ -1,37 +1,30 @@
 # Installing gems
 
 As mentioned in the 
-[`README`](https://github.com/graalvm/truffleruby/tree/truffle-head/README.md) 
+[`README`](https://github.com/graalvm/truffleruby/tree/master/README.md) 
 TruffleRuby currently cannot run `gem install` out of the box because of incomplete
 support for openssl and Nokogiri. However there is a workaround which can be used
 to get both `gem install` and `bundler install` working.
 
-First, make sure `GEM_HOME`, `GEM_PATH` and `GEM_ROOT` are not set
-as they would interfere with RubyGems/Bundler.
-You can do this by switching to the system Ruby
-with your Ruby version switcher if you have one:
-```bash
-rvm use system
-rbenv system
-chruby system
-```
-Otherwise, unset the variables with:
-```bash
-unset GEM_HOME GEM_PATH GEM_ROOT
-```
+> **Note:** Ensure you have ruby managers configured properly, see 
+[`doc/user/ruby-managers`](https://github.com/graalvm/truffleruby/blob/master/doc/user/ruby-managers.md).
```

Comment:
Users should not need to navigate too much to do one thing. It's also always best to be able to point to a single page and have clear prerequisites.